app/views/associate_ucr_confirmation.scala.html

```diff
@@ -14,32 +14,39 @@
  * limitations under the License.
  *@
 
+@import components.gds.gds_main_template
 @import controllers.routes
-@import uk.gov.hmrc.play.views.html._
 @import controllers.storage.FlashKeys
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 @import views.Title
 
-@this(main_template: views.html.main_template)
+@this(
+        govukLayout: gds_main_template,
+        govukPanel : GovukPanel,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @()(implicit request: Request[_], flash: Flash, messages: Messages)
 
 @prefix = @{ s"associate.${flash(FlashKeys.CONSOLIDATION_KIND)}.confirmation" }
-@associateLink = {<a href="@routes.ChoiceController.startSpecificJourney(forms.Choice.AssociateUCR.value)">@messages("association.confirmation.associateOrDepart.associate")</a>}
-@shutMucrLink = {<a href="@routes.ChoiceController.startSpecificJourney(forms.Choice.Departure.value)">@messages("association.confirmation.associateOrDepart.depart")</a>}
+@associateLink = {@components.gds.link(messages("association.confirmation.associateOrDepart.associate"), routes.ChoiceController.startSpecificJourney(forms.Choice.AssociateUCR.value))}
+@shutMucrLink = {@components.gds.link(messages("association.confirmation.associateOrDepart.shut"), routes.ChoiceController.startSpecificJourney(forms.Choice.ShutMUCR.value))}
```

Comment:
Here the messages are still `associateOrDepart` but you changed messages to be `associateOrShut`. You should change it here too.

Reply:
Good spot